test/sqllogictest/pg_get_viewdef.slt

```diff
@@ -0,0 +1,91 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# This file is derived from the logic test suite in CockroachDB. The
+# original file was retrieved on June 10, 2019 from:
+#
+# The original source code is subject to the terms of the Apache
+# 2.0 license, a copy of which can be found in the LICENSE file at the
+# root of this repository.
+
+statement ok
+CREATE TABLE t (a int, b int, c int)
+
+statement ok
+CREATE VIEW t_view AS SELECT a, b FROM t
+
+query T
+SELECT pg_get_viewdef('doesnotexist')
+----
+NULL
+
+query T
+SELECT pg_get_viewdef('t_view')
+----
+SELECT a, b FROM [u1 AS materialize.public.t]
```

Comment:
This is the syntax we get from calling `.to_string()` on a `Query` object. I think `u1` is an internal ID that we assign, it's not something specified by the user when creating the view. But it seems like the SELECT query still executes as expected, so emitting this syntax should be fine I guess?

Reply:
Ah, yeah--I don't think the implementation is quite as simple as storing the stringified query in the system table because that interferes with the system we've designed to handle renames.

If you look at `plan_show_create_view`, I believe we'll need to use a code path more akin to that. The bit that's missing here is constructing a `NameSimplifier` and using that to determine what the name of the embedded `GlobalId` is.

If you need any help weaving through the code, please let me know!

Reply:
Chatted with @teskje offline and I was being overly cautious about the ability for this to handle renames--actually handles it just fine.

Reply:
@teskje Oh! This should also use `to_ast_string_stable()`! Forgot to mention in the previous convo